<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mission</class>
 <widget class="QWidget" name="Mission">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QTreeWidget" name="treeWidget">
       <property name="indentation">
        <number>20</number>
       </property>
       <attribute name="headerDefaultSectionSize">
        <number>60</number>
       </attribute>
       <column>
        <property name="text">
         <string notr="true">Groups</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>ID</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Type</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Action</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Altitude</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Latitude</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Longitude</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Speed</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Course</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QComboBox" name="comboBox_action"/>
       </item>
       <item row="0" column="0">
        <widget class="QComboBox" name="comboBox_type"/>
       </item>
       <item row="1" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Alt</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_Alt">
           <property name="inputMask">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="clear_Alt">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>error.png</normaloff>error.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Lat</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_Lat"/>
         </item>
         <item>
          <widget class="QPushButton" name="clear_Lat">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../Parameter/error.png</normaloff>../Parameter/error.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Spd</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_Spd"/>
         </item>
         <item>
          <widget class="QPushButton" name="clear_Spd">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../Parameter/error.png</normaloff>../Parameter/error.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Lng</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_Lng"/>
         </item>
         <item>
          <widget class="QPushButton" name="clear_Lng">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../Parameter/error.png</normaloff>../Parameter/error.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="6" column="0">
        <widget class="QPushButton" name="pushButton_ok">
         <property name="text">
          <string>OK</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Cog</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_Cog"/>
         </item>
         <item>
          <widget class="QPushButton" name="clear_Cog">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../Parameter/error.png</normaloff>../Parameter/error.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="6" column="1">
        <widget class="QPushButton" name="pushButton_cancel">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QPushButton" name="pushButton_clearall">
         <property name="text">
          <string>clearall</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QPushButton" name="pushButton_upload">
         <property name="text">
          <string>upload</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QPushButton" name="pushButton_6">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QPushButton" name="pushButton_download">
         <property name="text">
          <string>download</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0" colspan="2">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="9" column="0">
        <widget class="QPushButton" name="pushButton_LoadFromFile">
         <property name="text">
          <string>LoadFromFile</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QPushButton" name="pushButton_SaveToFile">
         <property name="text">
          <string>SaveToFile</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>clear_Alt</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_Alt</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>459</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>425</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clear_Lat</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_Lat</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>443</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>403</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clear_Lng</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_Lng</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>440</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>418</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clear_Spd</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_Spd</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>433</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>417</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clear_Cog</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_Cog</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>440</x>
     <y>190</y>
    </hint>
    <hint type="destinationlabel">
     <x>424</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
